_test_unstructured_client/test__decorators.py

```diff
@@ -1,70 +1,98 @@
+from unstructured_client.models import shared
+from unstructured_client.models.errors import SDKError
 
 
-def get_api_key():
-    api_key = os.getenv("UNS_API_KEY")
-    if api_key is None:
-        raise ValueError("""UNS_API_KEY environment variable not set. 
-Set it in your current shell session with `export UNS_API_KEY=<api_key>`""")
-    return api_key
+FAKE_KEY = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 
 
 @pytest.mark.parametrize(
     ("server_url"),
     [
-        ("https://unstructured-000mock.api.unstructuredapp.io"), # correct url
+        ("https://unstructured-000mock.api.unstructuredapp.io"),  # correct url
         ("unstructured-000mock.api.unstructuredapp.io"),
         ("http://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("https://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
-    ]
+    ],
 )
 def test_clean_server_url_on_paid_api_url(server_url: str):
     client = UnstructuredClient(
         server_url=server_url,
-        api_key_auth=get_api_key(),
+        api_key_auth=FAKE_KEY,
+    )
+    assert (
+        client.general.sdk_configuration.server_url
+        == "https://unstructured-000mock.api.unstructuredapp.io"
     )
-    assert client.general.sdk_configuration.server_url == "https://unstructured-000mock.api.unstructuredapp.io"
 
 
 @pytest.mark.parametrize(
     ("server_url"),
     [
-        ("http://localhost:8000"), # correct url
+        ("http://localhost:8000"),  # correct url
         ("localhost:8000"),
         ("localhost:8000/general/v0/general"),
         ("http://localhost:8000/general/v0/general"),
-    ]
+    ],
 )
 def test_clean_server_url_on_localhost(server_url: str):
     client = UnstructuredClient(
         server_url=server_url,
-        api_key_auth=get_api_key(),
+        api_key_auth=FAKE_KEY,
     )
     assert client.general.sdk_configuration.server_url == "http://localhost:8000"
 
 
 def test_clean_server_url_on_empty_string():
     client = UnstructuredClient(
         server_url="",
-        api_key_auth=get_api_key(),
+        api_key_auth=FAKE_KEY,
     )
     assert client.general.sdk_configuration.server_url == ""
 
+
 @pytest.mark.parametrize(
     ("server_url"),
     [
         ("https://unstructured-000mock.api.unstructuredapp.io"),
         ("unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
-    ]
+    ],
 )
 def test_clean_server_url_with_positional_arguments(server_url: str):
     client = UnstructuredClient(
-        get_api_key(),
+        FAKE_KEY,
         "",
         server_url,
     )
-    assert client.general.sdk_configuration.server_url == "https://unstructured-000mock.api.unstructuredapp.io"
+    assert (
+        client.general.sdk_configuration.server_url
+        == "https://unstructured-000mock.api.unstructuredapp.io"
+    )
+
+
+def test_suggest_defining_url_if_401():
```

Comment:
This name is better. I find it helpful to add the `it_...` prefix to get the descriptive ball rolling. I would mention how it suggests because that's not typically a REST API can do (as opposed to a GUI). Maybe `test_it_issues_a_warning_to_suggest_defining_a_URL_on_a_401()`.